### PR TITLE
Fix PEP8 in boots

### LINKED ISSUE #872: PEP8 in boots
dipy/boots/__init__.py:2:1: W391 blank line at end of file
dipy/boots/resampling.py:3:1: E265 block comment should start with '# '
dipy/boots/resampling.py:5:11: E401 multiple imports on one line
dipy/boots/resampling.py:12:80: E501 line too long (113 > 79 characters)
dipy/boots/resampling.py:12:99: E231 missing whitespace after ','
dipy/boots/resampling.py:14:1: E302 expected 2 blank lines, found 1
dipy/boots/resampling.py:21:1: E302 expected 2 blank lines, found 1
dipy/boots/resampling.py:21:27: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:21:29: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:21:38: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:21:40: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:21:52: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:21:54: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:71:28: E231 missing whitespace after ','
dipy/boots/resampling.py:75:9: E265 block comment should start with '# '
dipy/boots/resampling.py:79:58: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:79:60: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:81:1: E302 expected 2 blank lines, found 1
dipy/boots/resampling.py:81:21: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:81:23: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:81:29: E203 whitespace before ','
dipy/boots/resampling.py:81:37: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:81:39: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:81:49: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:81:51: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:117:5: E265 block comment should start with '# '
dipy/boots/resampling.py:126:5: E265 block comment should start with '# '
dipy/boots/resampling.py:135:5: E265 block comment should start with '# '
dipy/boots/resampling.py:140:1: E302 expected 2 blank lines, found 1
dipy/boots/resampling.py:145:11: E271 multiple spaces after keyword
dipy/boots/resampling.py:147:1: E302 expected 2 blank lines, found 1
dipy/boots/resampling.py:163:75: E203 whitespace before ','
dipy/boots/resampling.py:164:63: E127 continuation line over-indented for visual indent
dipy/boots/resampling.py:165:75: E203 whitespace before ','
dipy/boots/resampling.py:166:63: E127 continuation line over-indented for visual indent
dipy/boots/resampling.py:167:43: E202 whitespace before ')'
dipy/boots/resampling.py:170:1: E302 expected 2 blank lines, found 1
dipy/boots/resampling.py:170:27: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:170:29: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:193:1: E302 expected 2 blank lines, found 1
dipy/boots/resampling.py:199:13: E201 whitespace after '('
dipy/boots/resampling.py:200:45: E202 whitespace before ')'
dipy/boots/resampling.py:202:1: E302 expected 2 blank lines, found 1
dipy/boots/resampling.py:209:12: E128 continuation line under-indented for visual indent
dipy/boots/resampling.py:212:1: E302 expected 2 blank lines, found 1
dipy/boots/resampling.py:212:29: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:212:31: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:212:41: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:212:43: E251 unexpected spaces around keyword / parameter equals
dipy/boots/resampling.py:266:28: E231 missing whitespace after ','
dipy/boots/resampling.py:266:43: E261 at least two spaces before inline comment
dipy/boots/resampling.py:266:44: E262 inline comment should start with '# '
dipy/boots/resampling.py:268:10: E711 comparison to None should be 'if cond is None:'
dipy/boots/resampling.py:275:9: E265 block comment should start with '# '
dipy/boots/resampling.py:276:40: E203 whitespace before ':'
dipy/boots/resampling.py:278:9: E265 block comment should start with '# '
dipy/boots/resampling.py:281:52: E261 at least two spaces before inline comment
dipy/boots/resampling.py:281:53: E262 inline comment should start with '# '
dipy/boots/resampling.py:285:59: E128 continuation line under-indented for visual indent
dipy/boots/resampling.py:287:1: E302 expected 2 blank lines, found 1
dipy/boots/resampling.py:290:1: E302 expected 2 blank lines, found 1

### PR DESCRIPTION
**Pull request by ghoshbishakh**
fixes #872

### CODE REVIEW

File: dipy/boots/resampling.py [hunk 1]
```diff
@@ -1,15 +1,21 @@
 #!/usr/bin/python
 
-#import modules
+# import modules
```
Comment (arokem) [inline dipy/boots/resampling.py]: Strangest comment I've seen. Would  you mind removing it?

File: dipy/boots/resampling.py [hunk 2]
```diff
@@ -160,14 +170,15 @@ def __calc_z0(x, p_0, statistic, eps, a_hat, sigma_hat):
         b_hat[i] = __tt_dot_dot(i, x, p_0, statistic, eps)
         tt_dot[i] = __tt_dot(i, x, p_0, statistic, eps)
     b_hat = b_hat / (2 * n**2)
-    c_q_hat = (__tt(x, (1 - eps) * p_0 + eps * tt_dot / (n**2 * sigma_hat)
-                                                              , statistic) +
-               __tt(x, (1 - eps) * p_0 - eps * tt_dot / (n**2 * sigma_hat)
-                                                              , statistic) -
-               2 * __tt(x, p_0, statistic) ) / eps**2
+    c_q_hat = (__tt(x, (1 - eps) * p_0 + eps * tt_dot /
+                    (n**2 * sigma_hat), statistic) +
```
Comment (arokem) [inline dipy/boots/resampling.py]: For clarity, I would enclose the things that go together in parentheses:
     
    (__tt(x, ((1 - eps) * p_0) +
            (eps * tt_dot / (n**2 * sigma_hat)), statistic ...

File: dipy/boots/resampling.py [hunk 3]
```diff
@@ -160,14 +170,15 @@ def __calc_z0(x, p_0, statistic, eps, a_hat, sigma_hat):
...
+               __tt(x, (1 - eps) * p_0 - eps * tt_dot /
+                    (n**2 * sigma_hat), statistic) -
```
Comment (arokem) [inline dipy/boots/resampling.py]: Same here

**Comment by arokem** [comment]
A couple of small things. Otherwise, looks good to me.

File: dipy/boots/resampling.py [hunk 4]
```diff
@@ -1,15 +1,21 @@
 #!/usr/bin/python
 
-#import modules
+# import modules
 import time
-import sys, os, traceback, optparse
+import sys
+import os
+import traceback
+import optparse
```
Comment (ghoshbishakh) [inline dipy/boots/resampling.py]: @arokem atually if you inspect the modules are not used in the entire code.
time, sys, os, traceback, optparse, also rom copy import copy, deepcopy
these are never used. So should they be cleaned up?

Comment (arokem) [inline dipy/boots/resampling.py]: Yes - good call. Please do remove . Running `pyflakes` (https://pypi.python.org/pypi/pyflakes) or `flake8` ( https://pypi.python.org/pypi/flake8) on this should also tell you about unused modules, so you can verify that you are not removing something necessary.

**Comment by ghoshbishakh** [comment]
@arokem I hope it is ok?

**Comment by arokem** [comment]
Yeah. Looks good to me. Anyone else want to take a look?

**Pull request merged**